test/autotest.py

```diff
@@ -543,8 +547,9 @@ def testRestart():
 
     #start user programs
     for cmd in cmds:
-      procs.append(launch(BIN+"dmtcp_launch "+cmd))
-      sleep(S*SLOW)
+      procs.append(runCmd(BIN+"dmtcp_launch "+cmd))
+    #wait and see if process will die shortly after checkpoint
```

Comment:
Actually, the comment goes with the `sleep()` statement on the following line.  But here also, I should move the comment and the sleep statement lower, to just after the `WAITFOR()` statement.  But in that case, the `sleep` duplicates the purpose of the `sleep()` stmt found insides the `for i in range(CYCLES):` loop a little bit lower.

So, in this cae, I propose that we delete this particular `sleep()` stmt.